### ISSUE TITLE
[Firefox 71] manualRowResize e2e test does not pass after browser update

### ISSUE THREAD

**Issue by aninde** [handsontable/handsontable#6601]
### Description
Unlucky one e2e test does not pass after browser update on Firefox 71. The same test passed on an earlier version of Firefox.

```
manualRowResize > handle position in a table positioned using CSS's `transform` > should display the handles in the correct position, with holder as a scroll parent
Expected 719 to be close to 720, 0.
```

### Your environment
* Browser Name and version: Firefox 71
* Operating System: macOS Catalina 10.15

**Comment by aninde** [comment]
This also occurs in the newer version of Firefox 72.

**Comment by aninde** [comment]
Also exists in version 7.4.0. @wojciechczerniak could we please fix it before releasing 7.4.0? I understand that this is not in the scope, but maybe if I ask very polite 🥺🥺

**Comment by aninde** [comment]
This also occurs in the newer version of Firefox 73.

**Comment by pnowak** [comment]
V8.0.0 in FF 79 - the same error.
```manualRowResize > handle position in a table positioned using CSS's `transform` > should display the handles in the correct position, with holder as a scroll parent
Expected 864 to be close to 865, 0.```

And a lot of others.

**Comment by aninde** [comment]
v.8.0.0 from `master` on latest `FF 80.0.1`. The bug is still present.

``manualRowResize > handle position in a table positioned using CSS's `transform` > should display the handles in the correct position, with holder as a scroll parent
Expected 953 to be close to 954, 0.``

**Comment by swistach** [comment]
Solved in #7295
